viper.go

```diff
@@ -1371,7 +1371,7 @@ func readAsCSV(val string) ([]string, error) {
 
 // mostly copied from pflag's implementation of this operation here https://github.com/spf13/pflag/blob/master/string_to_string.go#L79
 // alterations are: errors are swallowed, map[string]interface{} is returned in order to enable cast.ToStringMap
-func stringToStringConv(val string) interface{} {
+func stringToInterfaceCong(val string) interface{} {
```

Comment:
Typo: `Cong` vs `Conv`

Reply:
done

Reply:
Sorry, I'm not sure I was clear: it should be `Conv`

Reply:
sorry I mistakenly didn't pushed the changes